presto-server-rpm/pom.xml

```diff
@@ -97,6 +97,9 @@
                                 <dependency>
                                     <name>/usr/sbin/groupadd</name>
                                 </dependency>
+                                <dependency>
```

Comment:
is this cross portable among platforms (suse, centos)?

Reply:
i was testing this on centos6